### ISSUE TITLE
The LHCI role doesn't complete.

### ISSUE THREAD

**Issue by pm98zz-c** [codeenigma/ansible-provision#2049]
Trying to use the lhci role in ce-dev just stay stuck during provisioning. That seems to be due to https://github.com/codeenigma/ce-provision/blob/c4aeab06b708f07b7b88a05f577970a949a2dbf0/roles/debian/lhci/tasks/main.yml#L47 
There's another approach in the selenium role that could be reused here.

**Comment by gregharvey** [comment]
Here's some more background from an internal ticket:

----------------------

I think we're maybe starting it up wrong. Here's the task:

```yaml
- name: Configure Xvfb to start on boot.
  ansible.builtin.shell: |
    Xvfb -ac :99 -screen 0 1280x1024x16 &
```

* https://github.com/codeenigma/ce-provision/blob/1.x/roles/lhci/tasks/main.yml#L34C1-L36C42

Maybe a dev can have a play with running that actual command on a broken container to see what it does? Is there a display already running, do we need to do something else to ensure LHCI starts when the container/server does other than this? I'm already not convinced the task does what it thinks it does. Looks to me like that will start Xvfb once, but not on reboot.

Some info here, although it's quite old. I'm almost certain there's a better/smarter way to start Xvfb though.
* https://unix.stackexchange.com/questions/10465/how-do-i-start-xvfb-on-startup-on-debian

----------------------

Basically, what Pascal said! 😄 

And Chris added this:

----------------------

On `ce-dev provision`, the process seemed to hang. I hit CTRL-C, then `ce-dev shell <container-id>` and ran:


```
ce-dev@REDACTED-web:~$ Xvfb -ac :99 -screen 0 1280x1024x16 &
[1] 21101
ce-dev@REDACTED-web:~$ (EE) 
Fatal server error:
(EE) Server is already active for display 99
        If this server is no longer running, remove /tmp/.X99-lock
        and start again.
(EE)
```

At this point, it's waiting for stdin.

Maybe I can experiment with removing /tmp/.X99-lock before this.

**Comment by DionisioFG** [comment]
Selenium role has been re-written, and lighthouse can be executed in headless mode

https://github.com/codeenigma/ce-provision/issues/1079

**Comment by gregharvey** [comment]
Thanks Dioni, I see the code changes and I'm happy to merge them. I don't get what we need to do with the LHCI role to make this work though?
